packages/core/test/actions.test.ts

```diff
@@ -1246,6 +1246,19 @@ describe('forwardTo()', () => {
 
     service.send('EVENT', { value: 42 });
   });
+
+  it('should not cause an infinite loop when forwarding to undefined', () => {
+    const machine = createMachine({
+      on: {
+        '*': { cond: () => true, actions: forwardTo(undefined as any) }
+      }
+    });
+
+    const service = interpret(machine).start();
+
+    // This is testing for an infinite loop
+    service.send('TEST');
```

Comment:
And all should be well! Without adding that metadata, this would cause an infinite loop and this test would timeout (or worse, just hang).

This test completing is the success criterion.